src/uikit.cs

```diff
@@ -6265,14 +6759,15 @@ interface UIKeyCommand : NSCopying, NSSecureCoding {
 		NSString Escape { get; }
 
 		[iOS (9,0)]
+		[Deprecated (PlatformName.iOS, 13, 0, message: "Use 'UIKeyCommand' instead.")]
 		[Static]
 		[Export ("keyCommandWithInput:modifierFlags:action:discoverabilityTitle:")]
 		UIKeyCommand Create (NSString keyCommandInput, UIKeyModifierFlags modifierFlags, Selector action, NSString discoverabilityTitle);
 
 		[iOS (9,0)]
 		[NullAllowed]
 		[Export ("discoverabilityTitle")]
-		NSString DiscoverabilityTitle { get; set; }
+		NSString DiscoverabilityTitle { get; [Deprecated (PlatformName.iOS, 13, 0, message: "Use 'UIKeyCommand.DiscoverabilityTitle' instead.")] set; }
```

Comment:
This is `UIKeyCommand.DiscoverabilityTitle` ?!?
also nothing mention it's setter-only in the comment

Reply:
c/p will indeed destroy the world some day 🙈